e2e/e2e_test.go

```diff
@@ -493,7 +494,7 @@ var _ = Describe("Whereabouts functionality", func() {
 					err error
 				)
 
-				netAttachDef2 = macvlanNetworkWithWhereaboutsIPAMNetwork(testNetworkName+"2", testNamespace, ipv4TestRange, []string{}, testNetworkName+"2", false)
+				netAttachDef2 = macvlanNetworkWithWhereaboutsIPAMNetwork(testNetwork2Name, testNamespace, ipv4TestRange, []string{}, testNetwork2Name, false)
```

Comment:
please define this const in this `Context` block. It doesn't need to be available to no one else.